### PR TITLE
Changed the 'key' input value of the 'Get full detection details' task

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7188

## Description
The 'Get full detection details' task, which uses 'Set' automation, has an incorrect value for the 'key' input.
As the 'key' input was set to 'Detection.Behavior.pattern_disposition_details', all of the tasks within the playbook relate to 'Detection.Behavior'.

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Verify_Containment_Actions.yml [hunk 1]
```diff
@@ -784,8 +785,7 @@ outputs:
   description: Was the process contained?
   type: string
 tests:
-- No tests (auto formatted)
+- Test - CrowdStrike Falcon
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Verify_Containment_Actions.yml:788]: Pls remove this test

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### CrowdStrike Falcon Malware - Verify Containment Actions
+
+- Fixed an issue with the *'key'* input of the ***'Set'*** automation used within the **'Get full detection details'** task.
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md:6]:
```suggestion
Fixed an issue with the *'Get full detection details'* task's key.
```

Comment (TalNos) [reply Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md]: done

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Verify_Containment_Actions.yml (hunk 1 above)
Comment (TalNos) [reply Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Verify_Containment_Actions.yml]: done

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**